outplacement_completion_report_ipf_client/models/client_config.py

```diff
@@ -109,7 +109,7 @@ def get_url(self, path):
             url = self.url + '/' + path
         return url
 
-    def post_report(self):
+    def post_report(self, ):
```

Comment:
@AnatoliiYakymets  
Is this change necessary?

Reply:
Hello @anderswallenquist, This's the mistake because I wanted to change it to produce the ticket AFC 1486c.